hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/metrics2/util/SampleStat.java

```diff
@@ -117,7 +117,7 @@ public double total() {
    * @return  the arithmetic mean of the samples
    */
   public double mean() {
-    return numSamples > 0 ? (total / numSamples) : 0.0;
+    return numSamples > 0 ? a1 : 0.0;
```

Comment:
I am not sure why update here at HADOOP-13804. 
@xkrogen would you mind to give another check?